harness/determined/cli/workspace.py

```diff
@@ -459,6 +474,12 @@ def yaml_file_arg(val: str) -> Any:
                             help="existing namespace to which \
                             the workspace is bound",
                         ),
+                        cli.Arg("--auto-create-namespace", action="store_true", help="whether a \
+                            user wants a namespace auto-created and used for a \
+                            workspace-namespace binding."),
+                        cli.Arg("--auto-create-all-namespaces", action="store_true", help="whether \
+                            a user wants a namespace auto-created and used for each cluster's \
+                            respective workspace-namespace binding.")
```

Comment:
I might have missed it. Is `--auto-create-all-namespaces` being used anywhere?

Reply:
Yes! It's particularly being used in `api_workspace.go` to auto-create namespaces for all of the clusters being used. Ref below
```
if metadata.AutoCreateAllNamespaces {
	license.RequireLicense("auto-create namespace")
	autoCreatedNamespace, err := getAutoGeneratedNamespaceName(ctx, wkspID)
	if err != nil {
		return nil, err
	}
	namespace = *autoCreatedNamespace
	a.m.rm.CreateNamespace(*autoCreatedNamespace, clusterName, true)
} 
```